.project

```diff
@@ -8,4 +8,15 @@
 	</buildSpec>
 	<natures>
 	</natures>
+	<filteredResources>
```

Comment:
Is this really needed? If not: could you drop this? Looks to me like a configuration specific to your dev environment

Reply:
Done, sorry, I don't get, which is the second remark?